IntegrationTests/PurchaseTester/PurchaseTester/CatsViewController.swift

```diff
@@ -63,6 +68,28 @@ class CatsViewController: UIViewController {
     @objc func goPremiumButtonTapped() {
         navigationController?.popViewController(animated: true)
     }
+
+    @objc func manageSubButtonTapped() {
+        Purchases.shared.showManageSubscriptionModal { error in
+            print("Error opening Manage Sub Modal")
+        }
+    }
+
+    @objc func beginRefundButtonTapped() {
+        if #available(iOS 15.0, *) {
+            // refund for pro_cat's productID
+            Purchases.shared.beginRefundRequest(for: "com.revenuecat.purchaseTester.annual_39.99.2_week_intro")
```

Comment:
What do you think about moving this to `Constants.swift`?

Reply:
Maybe we can get this from the entitlement info instead? That way we don't have to hardcode it and it works with any API key

Reply:
yeah, i thought about that too, but our logic for even showing this button is based on the pro_cat entitlement. so that's a slightly larger refactor, happy to do it though

Reply:
there's not much that we can do about the hardcoded entitlement (maybe moving it to constants.swift would be nice), but for the product I'd really vote for refactoring it and getting it dynamically, so that it'll work at least for the same apps we've already got configured, which use the same entitlement but different product ids. 
Hardcoding the product means that this wouldn't work if you buy the monthly option instead, right? 

If we don't end up going with a dynamic product identifier, we should add a label with the hardcoded product identifier so that we know which product it's for when we're pressing the button, otherwise we might get confused as to why it's not working for all products